inf-terraform-azure/files/Makefile

```diff
@@ -0,0 +1,242 @@
+install-report:
+	awk '/Creation complete/ && !/terraform-data/ {print}' "$(INSTALL_REPORT_HOME)/tf_apply.log" > $(INSTALL_REPORT_HOME)/tf_created.log
+
+.PHONY: describe
+# Describe infrastructure.
+describe: init-terraform
+	TF_IN_AUTOMATION=1 TF_WORKSPACE="$(TF_WORKSPACE)" terraform output -json | tee outputs.json
+
+.PHONY: destroy
+# Destroy infrastructure.
+destroy: init-terraform
+destroy: export ARM_SUBSCRIPTION_ID := $(AZURE_SUBSCRIPTION_ID)
+destroy: export ARM_TENANT_ID       := $(AZURE_TENANT_ID)
+destroy: export ARM_CLIENT_ID       := $(AZURE_CLIENT_ID)
+destroy: export ARM_CLIENT_SECRET   := $(AZURE_CLIENT_SECRET)
+destroy: init-terraform
+	$(call check_defined,AZURE_SUBSCRIPTION_ID)
+	$(call check_defined,AZURE_TENANT_ID)
+	$(call check_defined,AZURE_CLIENT_ID)
+	$(call check_defined,AZURE_CLIENT_SECRET)
+
+	TF_IN_AUTOMATION=1 TF_WORKSPACE="$(TF_WORKSPACE)" terraform destroy -auto-approve
+
+.PHONY: install-dev-deps
+# Install development dependencies.
+install-dev-deps: install-git-pre-commit-hooks
+
+.PHONY: install-git-pre-commit-hooks
+# Install Git pre-commit hooks.
+install-git-pre-commit-hooks:
+	pre-commit install --overwrite
+
+.PHONY: install-ruby-gems
+# Install Ruby gems specified in Gemfile.
+install-ruby-gems:
+	BUNDLE_SILENCE_ROOT_WARNING=true bundle config --local path $(GEMS_HOME)
+	# see https://github.com/rubygems/rubygems/issues/4466 to get rid of error messages in Jenkins
+	BUNDLE_SILENCE_ROOT_WARNING=true TMPDIR=./vendor/tmp bundle install --jobs=8
+
+.PHONY: install-python-env
+# Install python virtual environment based on Pipfile
+install-python-env:
+	CI=true PIPENV_VENV_IN_PROJECT=true pipenv install
+
+.PHONY: init-terraform
+# Install Terraform workspace.
+init-terraform:
+	echo 1 | terraform init -backend=true -force-copy -input=false -backend-config="environments/$(TF_BACKEND).tfbackend.config"
+
+.PHONY: install-test-deps
+# Install testing dependencies.
+install-test-deps: install-ruby-gems install-python-env
+	# output azure account and user id for testing
+	az login --service-principal -u $(AZURE_CLIENT_ID) -p $(AZURE_CLIENT_SECRET) --tenant $(AZURE_TENANT_ID)
+	az account show --output tsv | tee $(INSTALL_REPORT_HOME)/azure_deploy_account.log
+
+.PHONY: cinc-auditor-test
+# run cinc-auditor without use of kitchen-terraform and create yaml for mapping terraform outputs to inspec inputs.
+cinc-auditor-test:
+	sh ./lib/scripts/createstackfixtureoutputs2yml.sh
+	bundle exec cinc-auditor exec test/integration/default --no-create-lockfile --no-distinct-exit --input-file ./test/integration/$$profile/files/inputs-from-tfo-stack.yml --target azure://
+
+# Checks if a variable is defined and produces a fatal error if not.
+check_defined = \
+	$(strip $(foreach 1,$1, \
+		$(call __check_defined,$1,$(strip $(value 2)))))
+__check_defined = \
+	$(if $(value $1),, \
+		$(error Undefined $1$(if $2, ($2))$(if $(value @), \
+			required by target `$@')))
+
+# Execute tests in included blueprints.
```

Comment:
anything following from here was removed

compare to https://github.com/opendevstack/ods-quickstarters/blob/master/inf-terraform-aws/files/Makefile#L156-L162